src/v4/PublicApi/Elements/CommonElements/TextElement.tsx

```diff
@@ -0,0 +1,32 @@
+import { TextProps, Text } from 'react-native';
+import React, { FC, memo } from 'react';
+import { ComponentID, ElementID, PageID } from '../../../enum/enumUIKitID';
+import useConfig from '../../../hook/useConfig';
+
+type TextElementType = Partial<TextProps> & {
+  pageID: PageID;
+  componentID: ComponentID;
+  elementID: ElementID;
+  text?: string;
+};
+
+const TextElement: FC<TextElementType> = ({
+  pageID = '*',
+  componentID = '*',
+  elementID,
+  text,
+  ...props
+}) => {
+  const { excludes, getConfig } = useConfig();
+  const configId = `${pageID}/${componentID}/${elementID}`;
+  const config = getConfig(configId);
+  const configText = config?.text ?? config?.title ?? '';
+  if (excludes.includes(configId)) return null;
+  return (
+    <Text testID={configId} accessibilityLabel={configId} {...props}>
+      {text ?? configText}
```

Comment:
This is reusable Element. 
This Element will be used in multiple places. e.g. in this case I use communityDisplayName which is coming from API and parent components.

Reply:
If we pass "ok" as text, we don't need to care configuration for text krub. Because that "ok" is coming from API data . 
e.g. why do we need to change communityDisplayName in configuration? . etc.